ctapipe/instrument/camera/readout.py

```diff
@@ -111,25 +111,11 @@ def from_name(cls, camera_name="NectarCam", version=None):
         else:
             verstr = f"-{version:03d}"
 
-        try:
-            tabname = "{camera_name}{verstr}.camreadout".format(
-                camera_name=camera_name, verstr=verstr
-            )
-            table = get_table_dataset(tabname, role="dl0.tel.svc.camera")
-            return CameraReadout.from_table(table)
-        except FileNotFoundError:
-            # TODO: remove case when files have been generated
-            logger.warning(
-                f"Resorting to default CameraReadout,"
-                f" File does not exist: ({tabname})"
-            )
-            reference_pulse_shape = np.array([norm.pdf(np.arange(96), 48, 6)])
-            return cls(
-                camera_name=camera_name,
-                sampling_rate=u.Quantity(1, u.GHz),
-                reference_pulse_shape=reference_pulse_shape,
-                reference_pulse_sample_width=u.Quantity(1, u.ns),
-            )
+        tabname = "{camera_name}{verstr}.camreadout".format(
+            camera_name=camera_name, verstr=verstr
```

Comment:
maybe use f-string syntax?

Reply:
I didn't add/change this code, just removed a try except around. There is no real reason to remove format